data_pipeline_api/registry/access_upload.py

```diff
@@ -403,45 +425,52 @@ def upload_model_run(
     type=click.Tuple([str, str]),
     help="(key, value) pairs that are passed to the remote storage, e.g. credentials",
 )
-@click.option("--remote-uri-override", type=str, help=f"URI to the root of the storage to post in the registry"
-                                                      f" required if the URI to use for download from the registry"
-                                                      f" is different from that used to upload the item")
-@click.option("--accessibility", type=str, default="public", help=f"accessibility of the data, defaults to public")
+@click.option(
+    "--remote-uri-override",
+    type=str,
+    help=f"URI to the root of the storage to post in the registry"
```

Comment:
Have you just gotten into the habit of making everything a format string? I got some low level warnings from pylint about this.

Reply:
copy + pasting of actual format strings...